ci/shell.nix

```diff
@@ -114,7 +114,7 @@ stdenvNoCC.mkDerivation ({
     ffmpeg
     dejavu_fonts
   ];
-  LD_LIBRARY_PATH = "${libffi}/lib:${libjpeg.out}/lib:${libusb1}/lib:${libressl.out}/lib";
+  DYLD_LIBRARY_PATH = "${libffi}/lib:${libjpeg.out}/lib:${libusb1}/lib:${libressl.out}/lib";
```

Comment:
Pushed by mistake, sorry.

Reply:
Anyway this is what is needed on MacOS to compile, maybe there should be an if or something.

Reply:
We can just use both if setting `LD_LIBRARY_PATH` doesn't cause problems on MacOS.